### ISSUE TITLE
What the coefficient λ in front of the MaskMI term in the loss functioncon？

### ISSUE THREAD

**Issue by PPBBJL** [Guanys-dar/MGDN#4]
Dear author, I can't find the coefficient λ in front of the MaskMI term in the loss function either in the paper or in the code. Please let me know, thank you very much for your contribution!

**Comment by Guanys-dar** [comment]
Hello, thank you for your interest. The weight setting for the MaskMI term mainly depends on the specific task. This is due to the inherent characteristics of different tasks. Generally speaking, a smaller training weight (such as 1e-3) will ensure a stable training process. You can adjust it according to your task's needs. If you have any further questions, feel free to contact me!
